docparse/output_label_options_walkthrough.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: 'Output Label Options'
+description: 'Applying heuristics to tweak the output'
+icon: 'toolbox'
+---
+
+Sometimes the output you get from Aryn DocParse isn't exactly what you want. You can mitigate this by specifying the `output_label_options` which will apply simple heuristics to correct the output. Currently the only heuristic we support is 
+`promote_title` which will check if there's no title in the output and then intelligently choose one of other elements in the output to promote to title.     
```

Comment:
Should we make sure to mention that this only occurs on the first page?

Maybe change it to say """Currently the only heuristic we support is `promote_title` which, when enabled, will check if there's no element with type "Title" on the first page and then intelligently choose one of other elements in the output to promote to title."""

Do we want to be very specific in our description here, or just give an overview? We could more explicitly explain how the element to be promoted to "Title" is chosen (it's the one with the largest font size among the elements on the first page that are of a type listed in the `title_candidate_elements` option).

For reference, [here](https://github.com/aryn-ai/sycamore/blob/680cea76c4505cdff141256c245101da86fb9a3c/lib/sycamore/sycamore/transforms/partition.py#L422-L430) and below is the updated docstring for this option.

```
output_label_options: A dictionary for configuring output label behavior. It supports two options:
    promote_title, a boolean specifying whether to pick the largest element by font size on the first page
        from among the elements on that page that have one of the types specified in title_candidate_elements
        and promote it to type "Title" if there is no element on the first page of type "Title" already.
    title_candidate_elements, a list of strings representing the label types allowed to be promoted to
        a title.
    Here is an example set of output label options:
        {"promote_title": True, "title_candidate_elements": ["Section-header", "Caption"]}
    default: None (no element is promoted to "Title")
```

Reply:
will address

Reply:
actually I go into more details below, so will fix this there.

Reply:
I am okay with not being very specific, but saying "which will check if there's no title in the output" gives users a misunderstanding of this feature. Let's make it say "which will check if there's no title on the first page".

Reply:
I see, thanks for the clarification. I'll add that.